encrypt/ibe/ibe_test.go

```diff
@@ -1,5 +1,6 @@
 package ibe
 
+/*
```

Comment:
I uncommented them and used the circl bls12381 instead but there is a difference between the scalar of circl bls and the one of the other pairing curve which makes it not pass the tests. I am trying to change the circl implementation so that it uses mod.Int

Reply:
The conversion is not needed if rejection sampling is updated as in: 
https://github.com/Robingoumaz/kyber-drand/pull/1